packages/scalecube-microservice/src/api/ProxiesMap.ts

```diff
@@ -0,0 +1,6 @@
+/**
+ * Map of generic proxyName and a Promise to the proxy
+ */
+export default interface ProxiesMap {
+  [proxyName: string]: Promise<any> | any;
```

Comment:
for readability better  Promise<msProxy> than Promise<any>
use type alias

Reply:
```suggestion
  [proxyName: string]: Promise<msProxy> | msProxy;

  type msProxy = any;
  
```